tests/exclude_system_edl/README.md

```diff
@@ -0,0 +1,7 @@
+# Exclude System EDL test
+
+This is a temporary test to ensure that an enclave application builds properly
+when the SDK is compiled with `-DCOMPILE_SYSTEM_EDL=OFF`.
+
+In the future all other tests should be updated to import system EDL in their own
```

Comment:
Is there a github issue # to track this?  Can we reference it in the comments?

Reply:
#2866